test/artifact/inspec_plugin_test.rb

```diff
@@ -0,0 +1,15 @@
+require_relative "artifact_helper"
+
+class TestInspecPlugin < Minitest::Test
+  parallelize_me!
+
+  def test_plugin
+    flunk
+    command = "/bin/inspec plugin #{TEST_CLI_OPTS}"
+    stdout, stderr, status = Open3.capture3(command)
+
+    assert_empty stderr.sub(/#< CLIXML\n/, "")
+    assert stdout
```

Comment:
I suspect that this line can't fail because `capture3` always returns a String. I don't think we want to test the contents tho, other than maybe being non-empty?

Reply:
Yes. In most cases I just want to make sure its not-empty. I generally don't care about the string contents here.